tests/lib/dirs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Default applies for: Ubuntu, Debian
+SNAPMOUNTDIR=/snap
+LIBEXECDIR=/usr/lib
+CORELIBEXECDIR=/usr/lib
```

Comment:
@zyga We name those variables else SNAPMOUNTDIR and LIBEXECDIR too, so this is just for consistency.